react-native/react/engine/index.js

```diff
@@ -118,8 +118,10 @@ class Engine {
   _generalIncomingRpc (method, param, response) {
     (this.generalListeners[method] || []).forEach(listener => {
       // TODO(mm) does it make sense to ever pass the response?
-      // It'll be weird if there are multiple listeners
-      listener(param)
+      // It'll be weird if there are multiple listeners.
+      // We should get the response callback to work the first
+      // time through and fail the second time.
+      listener(param, response)
```

Comment:
@chrisnojima I (accidentally) called response.result() twice and the service closed the connection with:
```
12:55:03.375891 ▶ [WARN keybase log.go:230] 015 {unix://@} Unexpected seqno 7 in incoming reply
12:55:03.375964 ▶ [ERRO keybase log.go:128] 016 {unix://@} Error in transport: Call not found for sequence number 7
```
So maybe your code isn't working?